rust/operator-binary/src/backend/tls/mod.rs

```diff
@@ -43,7 +43,7 @@ mod ca;
 
 /// How long CA certificates should last for. Also used for calculating when they should be rotated.
 /// Must be less than half of [`DEFAULT_MAX_CERT_LIFETIME`].
```

Comment:
I believe this comment is wrong, but I don't know what is should be replaced with.